### DISCUSSION TITLE
Templater Plugin: How to block link update/reference/backlink for all notes in my templates folder so that an unlinked link creates a new note every time

### DISCUSSION THREAD

**Discussion by Teighe** [SilentVoid13/Templater#1053] [Help]
I have a Daily Note template that includes these links to create a new note:
```
![[Journal/Dreams/New Note]]
![[Journal/Today I Learned/New Note]]
![[Journal/Reflections/New Note]]
```
When I create a new note using my Daily Note template, and then click on one of these links, it creates a new note in the corresponding folder. Because I have unique templates assigned to the individual folders, the names of the new note always change from "New Note" to a unique name that includes the date. So, "New Note.md" no longer exists and is replaced by something like "2023-03-13 Dream.md". I like that my daily note now references the existing with the new name, clicking the links in my individual daily note also updates the links in my Daily Note Template. 

What I want is for the path to stay preserved and unlinked in my Daily Note template so the next time I use it, the link is fresh and ready for me to create a new file. 

Is it possible to block whatever is happening there? Is it backlinking or tranclusion? I'm new to all of this. Thank you!

**Comment by Zachatoo** [comment]
In your template, instead of having a link, you can concatenate strings and generate the link when the template is applied. So your daily notes will still have updated links without updating the template, since the template doesn't actually have a link in it.

```js
<% "![[" + "Journal/Dreams/New Note" + "]]" %>
```
